tests/data/NewDatabase.xml

```diff
@@ -41,12 +41,12 @@
 		</Binaries>
 		<CustomData>
 			<Item>
-				<Key>A Sample Test Key</Key>
-				<Value>valu</Value>

```

Comment:
@ckieschnick looks like tabs are used in the rest of the file, maybe it would be better to keep it that way for consistency.